flink-connectors/flink-connector-kafka-0.10/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer010.java

```diff
@@ -69,6 +70,16 @@
 
 	private static final long serialVersionUID = 2324564345203409112L;
 
+	/** Configuration to set consumer prefix for ratelimiting. **/
+	private static final String CONSUMER_PREFIX = "kafka";
+
+	/** Default value for ratelimit flag. **/
+	private static final boolean DEFAULT_USE_RATELIMITING = false;
+
+	/** Default value for ratelimit which is default max.partition.fetch.bytes. **/
+	private static final long DEFAULT_MAX_BYTES_PER_SECOND = 1 * 1024 * 1024L;
```

Comment:
This actually makes for a very low default rate limit.  Would it better to default to something much higher?

Reply:
In fact I suggest no default.  If the feature is enabled and the rate limit isn't set it should throw an exception.

Reply:
I also suggest you move this default to RateLimitingConfig.